lib/resources.rb

```diff
@@ -1131,6 +1131,46 @@ def get_resource(key)
                 ]
                }]}
 
+    DSTU2_DIAGNOSTIC_REPORT ||= {
```

Comment:
Don't need a separate constant for a single report since we don't support retrieving by id